Code/GraphMol/Fingerprints/TopologicalTorsionGenerator.cpp

```diff
@@ -21,8 +21,8 @@ TopologicalTorsionArguments::TopologicalTorsionArguments(
     const bool includeChirality, const uint32_t torsionAtomCount,
     const bool countSimulation, const std::vector<std::uint32_t> countBounds,
     const std::uint32_t fpSize)
-    : FingerprintArguments(countSimulation, countBounds, fpSize),
-      df_includeChirality(includeChirality),
+    : FingerprintArguments(countSimulation, countBounds, fpSize, 1,
```

Comment:
Was this a bug from the initial review?

Reply:
Might have been hard to catch in the review, but I definitely should have removed this when I moved the `df_includeChirality` data member to the base class.